components/brave_ads/core/internal/legacy_migration/database/database_constants.h

```diff
@@ -10,8 +10,8 @@
 
 namespace brave_ads::database {
 
-constexpr int32_t kVersion = 25;
-constexpr int32_t kCompatibleVersion = 25;
+constexpr int32_t kVersion = 27;
```

Comment:
It would appear database changes do not reflect migration to version 27, can you please check all database changes. Thanks